enforcer/acls/acls.go

```diff
@@ -132,7 +132,7 @@ func (c *ACLCache) AddRuleList(rules policy.IPRuleList) (err error) {
 
 	err = nil
 	for _, rule := range rules {
-		if lerr := c.AddRule(rule); err != nil {
+		if lerr := c.AddRule(rule); lerr != nil {
 			err = lerr
```

Comment:
As a quick question: Do we mean to continue trying to add rules even if we encounter an error?